### DISCUSSION TITLE
Stay in the line.... bonus track!

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#81] [General]
A middle box is not same size as others : 

![Screenshot 2023-05-25 at 09 14 31](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/900ed818-0db1-4966-b508-5d2e191ac4b9)

**Comment by TJ-BitCanna** [comment]
Thanks for noticing! We are also looking into responsiveness of the webwallet and a new design overall which is in line with our mobile app which we are developing.

I'll get back on this after internal discussion what we'll do time-wise regarding design.
